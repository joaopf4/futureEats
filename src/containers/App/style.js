import styled from "styled-components";

export const AppWrapper = styled.div`
    height: 100vh;
    width: 100vw;
    display: flex;
    align-items: center;
`
export const DisplayWrapper = styled.div`
    border: 1px solid black;
    height: 667px;
    width: 375px; 
    margin: 0 auto;
`
export const StatusBar = styled.div`
    height: 20px;
    width: 100%;
    
`
export const PageWrapper = styled.div`
    height: 647px;
    width: 100%; 
`


