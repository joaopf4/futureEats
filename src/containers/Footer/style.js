import styled from "styled-components";


export const FooterWrapper = styled.div`
    height: 49px;
    width: 100%;
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    align-items:center;
    justify-items: center;
    box-shadow: 0 -0.9px 0 0 rgba(0, 0, 0, 0.25);
    z-index: 1;

    svg{
        fill: #b8b8b8;
    }

    svg:nth-child(${props=>props.active}){
        fill: #5cb646;
    }
`