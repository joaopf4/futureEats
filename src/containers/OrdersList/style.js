
import styled from 'styled-components'
import Typography from '@material-ui/core/Typography'

export const OrdersListWrapper = styled.div`
width: 100%;
height: ${({orders}) => (orders ? '50' : 'auto')};
align-items: center;
display: flex;
flex-direction: column;
padding: 12px 4px 0 16px;
`

export const IfEmpty = styled(Typography)`
text-align: center;
font-size: 16px;
padding: 12px 0;
`

export const OrderWrapper = styled.div`
width: 100%;
height: ${({orders}) => (orders ? '42' : '200px')};
overflow-y: scroll;
`

export const RestaurantInfo = styled.div`
width: 100%;
height: 86px;
padding-bottom: 8px;
display: flex;
flex-direction: column;
justify-content: space-evenly;
`

export const RestaurantName = styled(Typography)`
font-size: 16px;
font-weight: 700;
color: #5cb646;
`

export const RestaurantAddress = styled(Typography)`
font-size: 16px;
color: #b8b8b8;
`

export const ExpectedTime = styled(Typography)`
font-size: 16px;
color: #b8b8b8;
`
