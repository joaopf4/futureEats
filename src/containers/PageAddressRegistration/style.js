import styled from 'styled-components'
import Typography from '@material-ui/core/Typography'
import TextField from '@material-ui/core/TextField'
import Button from '@material-ui/core/Button'
import { FaAngleLeft } from 'react-icons/fa'

export const Form = styled.form`
align-items: center;
display: flex;
justify-content: center;
flex-direction: column;
`
export const LoginWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: flex-start;
`
export const AdressHeader = styled.div `
    height: 44px;
    width: 100%;
    border-bottom: 1px #b8b8b8 solid;
    padding-left: 16px ;
`
export const MyAddress = styled(Typography)`
    margin-top: 17px;
    padding: 5px 20px 5px 20px;
    font-size: 16px;
    font-weight:  bold;
`
export const Inputs = styled(TextField)`
    margin-top: 16px;
    width: 328px;
    height: 56px;
`
export const SaveButton = styled(Button)`
    width: 328px;
    height: 42px;
    border-radius: 2px;
    margin-top: 16px;
`   
export const BackIcon = styled(FaAngleLeft) `
    margin-top: 12px;
    :active{
    background-color: #b8b8b8;
    border-radius: 6px;
}
`



