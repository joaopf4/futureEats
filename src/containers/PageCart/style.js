import styled from 'styled-components'
import Typography from '@material-ui/core/Typography'
import Button from '@material-ui/core/Button'
import RadioGroup from '@material-ui/core/RadioGroup'
import FormControlLabel from '@material-ui/core/FormControlLabel'
import Radio from '@material-ui/core/Radio';


export const CartWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: flex-start;
    max-height: 100%;
`

export const Title = styled(Typography)`
    width: 100%;
    font-size: 19px;
    padding: 12px 0;
    text-align: center;
    `

export const DeliveryAddressWrapper = styled.div`
    width: 100%;
    height: 76px;
    padding: 16px 16px;
    background-color: #eeeeee;
    justify-content: space-around;
    display: flex;
    flex-direction: column;
    box-shadow: 0 -0.9px 0 0 rgba(0, 0, 0, 0.25);
`

export const AddressTitle = styled(Typography)`
    font-size: 16px;
    color: #b8b8b8;
    width: 328px;
`

export const Address = styled(Typography)`
    font-size: 16px;
    width: 328px;
`

export const PaymentWrapper = styled.div`
    min-height: ${({orders}) => (orders ? 'none' : '355px')};
    max-height: 100%;
    height: auto;
    width: 100%;
    display: flex;
    flex-direction: column;
    padding: 16px 16px 0 16px;
`

export const DeliveryPrice = styled(Typography)`
    width: 100%;
    font-size: 16px;
    text-align: end;
`

export const TotalPrice = styled.div`
    display: flex;
    width: 100%;
    padding: 14px 0;
`

export const SubTotalTitle = styled(Typography)`
    font-size: 18px;
    width: 50%;
    text-align: start;
`

export const SubTotalPrice = styled(Typography)`
    font-size: 18px;
    width: 50%;
    text-align: end;
    font-weight: 700;
    color: #5cb646;
`

export const PaymentMethod = styled.div`
    width: 100%;
`

export const PaymentTitle = styled(Typography)`
    width: 100%;
    padding: 8px 0;
    border-bottom: solid 1px #000000;
`

export const PaymentRadioGroup = styled(RadioGroup)`
    height: auto;
    width: 100%;
    padding-top: 11px;
`

export const RadioPayment = styled(FormControlLabel)`
    width: 100%;
    height: 18px;
    padding: 14px 0;
    font-size: 16px;
    color: black;
`
export const RadioCustom = styled(Radio)`
    && {
        color: #000000
    }
`
export const ConfirmButton = styled(Button)`
    width: 328px;
    height: 42px;
    border-radius: 2px;
    margin: 14px 0;
    background-color: #5cb646;

    &&:hover{
        background-color: #5cb646;
    }
`   