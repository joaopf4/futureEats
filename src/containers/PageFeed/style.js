import styled from "styled-components";
import Typography from "@material-ui/core/Typography";


export const FeedWrapper = styled.div`
    height: 100%;
    width: 100%;
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 44px 72px 42px 1fr 49px;
    justify-items:center;
    align-items:center;
`

export const LogoWrapper = styled.div`
    border-bottom: 1px solid #b8b8b8;
    height: 100%;
    width: 100%;
    display: flex;
    justify-content: center;
`

export const Logo = styled(Typography)`
    margin: auto;
    font-size: 16px;
`

export const NavBar = styled.div`
    height: 42px;
    width: 100%;
    display: flex;
    align-items: center;
    overflow-x: scroll;
    padding-left: 8px;
`

export const OptionText = styled(Typography)`
    padding: 8px;
    font-size: 18px;

    :nth-of-type(${props => props.activeOption}){
       && {
         color: green;  
       } 
    }
`

export const ContentWrapper = styled.div`
    width: 100%;
    height: 100%;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    overflow-y: scroll;
`