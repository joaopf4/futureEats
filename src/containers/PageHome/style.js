import styled from "styled-components"



export const HomeWrapper = styled.div`
    background-color: black;
    width: 100%;
    height: 100%;
    display: flex;
    justify-content: center;
    align-items: center;

`

export const Logo = styled.img`
  width: 126px;
  height: 65px;
`
