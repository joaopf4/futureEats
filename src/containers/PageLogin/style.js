import styled from 'styled-components'
import Typography from '@material-ui/core/Typography'
import TextField from '@material-ui/core/TextField'
import Button from '@material-ui/core/Button'


export const LoginWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: flex-start;
    padding: 0 16px 0 16px;/* acrescentei esse padding lateral */
`

export const Logo = styled.img`
  width: 104px;
  height: 58px;
  margin-top: 88px;

`
export const Entrar = styled(Typography)`
    margin-top: 28px;
    padding: 5px 20px 5px 20px;
    font-size: 16px;
    font-weight: bold;
`
export const Inputs = styled(TextField)`
    margin-top: 16px;
    width: 100%;/* mudei pra 100% */
`
export const EntrarButton = styled(Button)`
     width: 100%;/* mudei pra 100% */
    height: 42px;
    border-radius: 2px;
    margin-top: 16px;
    
`   
export const Cadastrar = styled(Typography)`
margin-top: 28px;
padding: 5px 20px 5px 20px;
font-size: 16px;
`
export const Form = styled.form`
align-items: center;
display: flex;
justify-content: center;
flex-direction: column;
`

export const Span = styled.span`
cursor: pointer;
:hover{
    color: blue;
    text-decoration: underline;
}
`