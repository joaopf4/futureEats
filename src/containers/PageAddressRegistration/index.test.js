import React from 'react'
import {shallow} from 'enzyme'
import { Form, LoginWrapper, AdressHeader, BackIcon, Logo, MyAddress, Inputs, SaveButton } from './style';
import {PageAddressRegistration} from './index'
import renderer from 'react-test-renderer'



describe ("Testing PageAddressRegistration", () => {
    // test("Finding subComponents", () => {
    //     //preparação
    //     const renderedComponent = shallow(<PageAddressRegistration/>)
    //     const findLoginWrapper = renderedComponent.find(LoginWrapper)
    //     const findLogo = renderedComponent.find(Logo)
    //     const findMyAddress = renderedComponent.find(MyAddress)
    //     const findInputs = renderedComponent.find(Inputs)
    //     const findSaveButton = renderedComponent.find(SaveButton)

    //     //verificação
    //     expect(findLoginWrapper).toHaveLength(1);
    //     expect(findLogo).toHaveLength(1);
    //     expect(findMyAddress).toHaveLength(1);
    //     expect(findInputs).toHaveLength(6);
    //     expect(findLogo).toHaveLength(1);
        
    // })
    // test("handleInputChange", () => {
    //     //preparação
    //     const renderedComponent = shallow(<PageAddressRegistration/>)
    //     const findStreet = renderedComponent.find({name: "street"})
    //     const findNumber = renderedComponent.find({name: "number"})
    //     const findNeighbourhood = renderedComponent.find({name: "neighbourhood"})
    //     const findCity = renderedComponent.find({name: "city"})

    //     //execução
    //     findStreet.simulate("change", {target: {value: "test", name: "street"}})
    //     findNumber.simulate("change", {target: {value: "test1", name: "number"}})
    //     findNeighbourhood.simulate("change", {target: {value: "test2", name: "neighbourhood"}})
    //     findCity.simulate("change", {target: {value: "test3", name: "city"}})
      
    //     //verificação
    //     expect(renderedComponent.state().street).toEqual("test")
    //     expect(renderedComponent.state().number).toEqual("test1")
    //     expect(renderedComponent.state().neighbourhood).toEqual("test2")
    //     expect(renderedComponent.state().city).toEqual("test3")
        
    // })
    test("Testando styled do Form", () => {
        const newSnapShot = renderer.create(<Form />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do LoginWrapper", () => {
        const newSnapShot = renderer.create(<LoginWrapper />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do AdressHeader", () => {
        const newSnapShot = renderer.create(<AdressHeader />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do MyAddress", () => {
        const newSnapShot = renderer.create(<MyAddress />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do Inputs", () => {
        const newSnapShot = renderer.create(<Inputs />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do SaveButton", () => {
        const newSnapShot = renderer.create(<SaveButton />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })
    test("Testando styled do BackIcon", () => {
        const newSnapShot = renderer.create(<BackIcon />).toJSON();

        expect(newSnapShot).toMatchSnapshot();
    })

})
