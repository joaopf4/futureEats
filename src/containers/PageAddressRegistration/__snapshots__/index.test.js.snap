// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PageAddressRegistration Testando styled do AdressHeader 1`] = `
<div
  className="sc-htpNat hFenuF"
/>
`;

exports[`Testing PageAddressRegistration Testando styled do BackIcon 1`] = `
<svg
  className="sc-bZQynM ggdgFL"
  fill="currentColor"
  height="1em"
  stroke="currentColor"
  strokeWidth="0"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 256 512"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
  />
</svg>
`;

exports[`Testing PageAddressRegistration Testando styled do Form 1`] = `
<form
  className="sc-bdVaJa jTWtvb"
/>
`;

exports[`Testing PageAddressRegistration Testando styled do Inputs 1`] = `
<div
  className="MuiFormControl-root-37 sc-ifAKCX hrNALZ"
>
  <div
    className="MuiInputBase-root-54 MuiInput-root-41 MuiInput-underline-45 MuiInputBase-formControl-55 MuiInput-formControl-42"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-64 MuiInput-input-49"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`Testing PageAddressRegistration Testando styled do LoginWrapper 1`] = `
<div
  className="sc-bwzfXH hkytMV"
/>
`;

exports[`Testing PageAddressRegistration Testando styled do MyAddress 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-bxivhb gDOFHL"
/>
`;

exports[`Testing PageAddressRegistration Testando styled do SaveButton 1`] = `
<button
  className="MuiButtonBase-root-97 MuiButton-root-71 MuiButton-text-73 MuiButton-flat-76 sc-EHOje kNLaCc"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-72"
  />
  <span
    className="MuiTouchRipple-root-100"
  />
</button>
`;
